samples/snippets/pom.xml

```diff
@@ -40,31 +40,18 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
   </properties>
 
-  <!-- [START pubsublite_install_with_bom] -->
-  <!-- [START pubsublite_java_dependencies] -->
-  <!--  Using libraries-bom to manage versions.
-  See https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/The-Google-Cloud-Platform-Libraries-BOM -->
-  <dependencyManagement>
-    <dependencies>
-      <dependency>
-        <groupId>com.google.cloud</groupId>
-        <artifactId>libraries-bom</artifactId>
-        <version>5.3.0</version>
-        <type>pom</type>
-        <scope>import</scope>
-      </dependency>
-    </dependencies>
-  </dependencyManagement>
-
   <dependencies>
+    <!-- [START pubsublite_install_without_bom] -->
+    <!-- [START pubsublite_java_dependencies] -->
     <dependency>
       <groupId>com.google.cloud</groupId>
-      <artifactId>google-cloud-pubsub</artifactId>
+      <artifactId>google-cloud-pubsublite</artifactId>
+      <version>0.1.4</version>
     </dependency>
     <dependency>
       <groupId>com.google.cloud</groupId>
-      <artifactId>google-cloud-pubsublite</artifactId>
-      <version>0.1.4</version><!-- {x-version-update:google-cloud-pubsublite:released} -->
+      <artifactId>google-cloud-pubsub</artifactId>
+      <version>1.105.0</version>
```

Comment:
I assume there is not need for this tag `<!-- {x-version-update:google-cloud-pubsublite:released} -->`?

Reply:
The released version will be updated by renovate bot after it's available in Maven Central. Otherwise, these tests will fail immediately after a release because the released version is not found

Reply:
I see now. Thanks!